demo-data/src/main/java/org/jboss/pnc/demo/data/DatabaseDataInitializer.java

```diff
@@ -614,12 +631,30 @@ public void initiliazeProjectProductData() {
 
     }
 
+    @TransactionAttribute
```

Comment:
Do you need this? You're creating your own transactions in helper methods.

Reply:
Resolved by call & conversation with Honza afterwards => Summary: we can let it here